### PR TITLE
make sure that if we send pixels in the new format they work too

### PR DESCRIPTION
**Pull request by steve8708**

### CODE REVIEW

File: packages/sdks/overrides/react-native/src/functions/transform-block.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import type { BuilderBlock } from '../types/builder-block';
 export function transformBlock(block: BuilderBlock): BuilderBlock {
   // Map the DOM-based pixel format to a native compatible one.
   // react-native doesn't support DOM specific fields like `tagName` and `properties`, but any browser framework (e.g. Vue, etc.) do
-  if (block.id.startsWith('builder-pixel-')) {
+  if (block.id.startsWith('builder-pixel-') && !block.component) {
```
Comment (samijaber) [inline packages/sdks/overrides/react-native/src/functions/transform-block.ts:6]: Could you expand the comment above the code to explain why this check is needed? It isn't clear to me at face-value

**Review by samijaber — APPROVED** [review]

File: packages/sdks/overrides/react-native/src/functions/transform-block.ts (hunk 1 above)
Comment (steve8708) [inline packages/sdks/overrides/react-native/src/functions/transform-block.ts:6]:
```suggestion
  // Additionally, we only want to convert the DOM pixel format (looks like `{ tagName: 'img', properties: { src: '...' } }`)
  // but we may move to a DOM-less pixel format that uses `{ component: { name: 'Image', ... } }` so if a `component` property is provided
  // we assume this accounts for native as well and we should not transform this block
  if (block.id.startsWith('builder-pixel-') && !block.component) {
```

Comment (steve8708) [reply packages/sdks/overrides/react-native/src/functions/transform-block.ts:6]: @samijaber better?

Comment (samijaber) [reply packages/sdks/overrides/react-native/src/functions/transform-block.ts:6]: Yes, perfect. Thanks 👍🏽

**Pull request merged**